src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -87,9 +103,21 @@ internal static TimeSeriesOperation ParseForBulkInsert(BlittableJsonReaderObject
                     return null; //never hit
                 }
 
+                long time = (long)bjro[0];
```

Comment:
This cannot be LazyNumberValue?

Reply:
LazyNumberValue has an implicit case to `long`

Reply:
But that won't trigger here. That is something that the _compiler_ inserts, not runtime.